ci_framework/roles/ci_metallb/tasks/metallb_k8s_install.yml

```diff
@@ -0,0 +1,105 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Create the metallb namespace
+  kubernetes.core.k8s:
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit)}}"
+    context: "{{ cifmw_openshift_context | default(omit) }}"
+    name: "{{ cifmw_ci_metallb_namespace }}"
+    kind: Namespace
+    state: present
+
+- name: Save k8s metallb OLM manifests as artifacts
+  ansible.builtin.copy:
+    dest: "{{ cifmw_ci_metallb_manifests_dir }}/metallb-{{ item.kind | lower }}-olm.yaml"
+    content: "{{ item | to_nice_yaml }}"
+  loop:
+    - "{{ cifmw_ci_metallb_olm_operator_group }}"
+    - "{{ cifmw_ci_metallb_olm_subscription }}"
+  loop_control:
+    label: "{{ item.metadata.name }}"
+
+- name: Create the OLM subscription resources
+  kubernetes.core.k8s:
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit)}}"
+    context: "{{ cifmw_openshift_context | default(omit) }}"
+    definition: "{{ item }}"
+    state: present
+  loop:
+    - "{{ cifmw_ci_metallb_olm_operator_group }}"
+    - "{{ cifmw_ci_metallb_olm_subscription }}"
+  loop_control:
+    label: "{{ item.metadata.name }}"
+
+- name: Wait for metallb operator to be intalled
+  kubernetes.core.k8s_info:
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit)}}"
+    context: "{{ cifmw_openshift_context | default(omit) }}"
+    namespace: "{{ cifmw_ci_metallb_namespace }}"
+    kind: ClusterServiceVersion
+    wait_sleep: 10
```

Comment:
Are we sure about this?
From the docs:
``` 
Whether to wait for certain resource kinds to end up in the desired state.

By default the module exits once Kubernetes has received the request.

Implemented for state=present for Deployment, DaemonSet and Pod, and for state=absent for all resource kinds.

For resource kinds without an implementation, wait returns immediately unless wait_condition is set.
```
Anyway, `wait` needs to be set to true.

Reply:
I think you're right @pablintino, it seems that csv do not have a condition type to be used with wait_condition, so I reverted it back to the original version with a command, thanks